sycl/source/detail/device_info.hpp

```diff
@@ -969,7 +969,11 @@ template <> struct get_device_info<bool, info::device::usm_device_allocations> {
         dev, pi::cast<RT::PiDeviceInfo>(info::device::usm_device_allocations),
         sizeof(pi_usm_capabilities), &caps, nullptr);
 
-    return (Err != PI_SUCCESS) ? false : (caps & PI_USM_ACCESS);
+    if (Err != PI_SUCCESS) {
+      Plugin.reportPiError(Err, "piDeviceGetInfo()");
```

Comment:
I think we need not throw the error to the user; just returning "false" for this query? 
Same comment in other queries where we can "deal" with the error.

Reply:
will restore the old code here.

Reply:
done

Reply:
done.